### PR TITLE
Make app top directory in zip

### PR DESCRIPTION
**Pull request by cmangla**
`timelapse.zip` contains `dist/Timelapse.app`, which might confuse some end users, so might be better to have `Timelapse.app` in the `.zip` without the `dist` directory.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by cmangla** [comment]
@mre How do I test this?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,12 +35,15 @@ jobs:
       run: poetry run python setup.py py2app --emulate-shell-environment
 
     - name: Compress app
-      run: zip -r timelapse.zip dist/Timelapse.app
+      shell: bash
+      run: |
+        ( cd dist; zip -r timelapse.zip Timelapse.app )
```
Comment (mre) [inline .github/workflows/ci.yml]: `bash` should be the default, no? Also I'm not sure why we need the parentheses. Isn't this equivalent?

```
    - name: Compress app
      run: |
        cd dist
        zip -r timelapse.zip Timelapse.app
        mv dist/timelapse.zip ./
```

**Comment by mre** [comment]
That's a great idea!

> @mre How do I test this?

Don't think you can. There used to be a tool for that, but it wasn't updated for the new YAML syntax yet. https://stackoverflow.com/questions/59241249/how-to-run-github-actions-locally
I usually test the changes in a branch and then squash the changes once I'm happy, but for releases that's tricky. 
I'd suggest we merge this as soon as we're happy with it and then see what happens? 😅

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cmangla) [reply .github/workflows/ci.yml]: I didn't know which shell was default and I wasn't sure if the shell environment was preserved across different lines in run (like a typical shell session), so this is just very conservative code. I'll change it to your suggestion just now.

**Comment by cmangla** [comment]
> I'd suggest we merge this as soon as we're happy with it and then see what happens? 😅

@mre Cool, I'll try that now then :)

**Comment by mre** [comment]
It looks like we're good! The pipeline didn't 💥, so maybe you can bump the version number to 0.996 and create a release at https://github.com/mre/timelapse/releases. 🎉 
If that's working as planned, the next step is to tag 1.0 and announce it. 
Kinda crazy to think that this app was almost 10 years in the making... 😆 There should be cake.

**Pull request merged**